### DISCUSSION TITLE
Is there a way to return PDF Page without tables

### DISCUSSION THREAD

**Discussion by danltw** [jsvine/pdfplumber#878] [Q&A] [answered]
Hi, I am currently trying to extract tables as a separate entity from the PDF Page class. My idea is that I already have some functions written using `page.extract_tables` and did some processing on the format to get what I want. However, I also want to continue using the page data/texts without the tables that I have extracted. Is there or has anyone done a solution where:

Pseudo-code:
- `page.extract_text`
- Identify tables
- Process tables
- Then remove tables from page object <-- struggling at this step
- return page object

Thanks in advance :)

**Comment by jsvine** [comment, marked as answer]
Hi @danltw, does the following comment help?: https://github.com/jsvine/pdfplumber/issues/242#issuecomment-668448246
